plugin/src/test/java/io/spine/tools/gradle/bootstrap/func/SpineBootstrapPluginTest.java

```diff
@@ -58,8 +59,8 @@
     private Path projectDir;
 
     @BeforeEach
-    void setUp(@TempDir Path dir) {
-        this.projectDir = dir;
+    void setUp() {
+        this.projectDir = TempDir.forClass(SpineBootstrapPluginTest.class).toPath();
```

Comment:
Please use [JUnit approach](https://junit.org/junit5/docs/current/user-guide/#writing-tests-built-in-extensions-TempDirectory).